test/language/block-scope/syntax/redeclaration-in-block/attempt-to-redeclare-function-declaration-with-function-declaration.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) Copyright 2011 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: B.3.3
+description: >
+    redeclaration within block:
+    attempt to redeclare function declaration with function declaration
+negative: SyntaxError
```

Comment:
I can see this being true in strict mode but is it also true in sloppy? Fairly surprising.